tests/basic.t

```diff
@@ -65,49 +65,94 @@ if [ $? -eq 1 ]; then
   TEST glusterd
 fi
 
-# Create gluster volume
+##### Create Gluster Volume Test Start #####
 TEST gluster vol create ${VOLNAME} ${HOST}:${BRKDIR} force
 
 # Start the volume
 TEST gluster vol start ${VOLNAME}
+##### End #####
 
 
-# Test gfapi access
+##### Gfapi Access Test Start #####
 TEST ./tests/gfapi-test ${VOLNAME} ${HOST}
+##### End #####
 
-# Start gluster-blockd.service
+
+##### Restart gluster-blockd.service Test Start #####
 systemctl daemon-reload
-TEST systemctl restart gluster-blockd.service
-sleep 1;
+TEST systemctl restart gluster-block-target.service gluster-blockd.service
```

Comment:
Why do we need to specify gluster-block-target.service ?

Reply:
I am okay to remove this.
I thought it also part of the gluster-block project.

Reply:
Yeah, gluster-block should internally take care of triggering gluster-block-target, so no need to explicitly mention it here.

Thanks!

Reply:
yeah, sure, will remove it.